### DISCUSSION TITLE
Failed to sign in to Renovate Dashboard

### DISCUSSION THREAD

**Discussion by 417-72KI** [renovatebot/renovate#11286] [Request Help]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Please select which platform you are using if self-hosting.

github.com

### Describe the bug

On https://app.renovatebot.com/dashboard, I select `Sign in with GitHub`.
Though GitHub authorization has been succeeded, it didn't transit to dashboard page.

### Relevant debug logs

<details><summary>Logs</summary>

```
Access to XMLHttpRequest at 'https://v1.renovateapi.com/github/user' from origin 'https://app.renovatebot.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
There is a terms and conditions page you need to accept on the first sign in. Maybe try in another browser or enable developer tools to look for errors.

**Comment by DawidJanczak** [comment]
We're running into the same issue and no T&Cs are visible on the initial sign in. Tested on different browsers and different machines. After initial sign in with Github the sign-in popup disappears, but nothing else changes. Clicking on the sign in button pop ups a window again, but that immediately closes and again nothing happens. We're using 2FA if that makes any difference :)

**Comment by rarkins** [comment]
What does the browser tools logs say? e.g. are there any 404s?

**Comment by rarkins** [comment]
Here's an example from a successful login. Note that I have checked the "Preserve log" box so that it records all network activity even after redirects:

![image](https://user-images.githubusercontent.com/6311784/129514161-d137812b-a952-406e-9a9b-c41bdecabd54.png)

**Comment by rarkins** [comment]
Something else you can try is manually going to https://app.renovatebot.com/register and submit the form

**Comment by DawidJanczak** [comment]
@rarkins I don't know if you changed anything, but it's working now. Thanks!

**Comment by esetnik** [comment]
After logging out of renovate and attempting to log back in I am also unable to login. The error is:

```
dashboard:1 Access to XMLHttpRequest at 'https://v1.renovateapi.com/github/user' from origin 'https://app.renovatebot.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by rarkins** [comment]
Sorry for the troubles on this. It seems intermittent so hard to work out what's going wrong. And *of course* I can never reproduce it myself no matter how many different times I log in/out or change browsers

**Reply by rarkins** [reply]
Switching this to a Discussion because there's no code in this repo which could possibly fix it

**Comment by raspy8766** [comment]
Encountering the same issue here. It was working just fine last Friday, then mid-day it stopped loading the logs. I tried to logout/login and it failed to authenticate with the same error:
![image](https://user-images.githubusercontent.com/8533429/129640351-2c83fa32-6eb1-4497-9cf0-b98016d1b6d3.png)

Was hoping it was just a fluke or temporary outage so I just tried it again today, still the same issue. Only change I can think of that may affect this is the org recently switched to [SAML auth with github](https://docs.github.com/en/organizations/managing-saml-single-sign-on-for-your-organization/enabling-and-testing-saml-single-sign-on-for-your-organization) earlier that week. Not sure if that could be related or if it's just a red herring.

Also tried the https://app.renovatebot.com/register page and I get the same error on form submission

**Comment by 417-72KI** [comment]
I took a failure log as similar as @rarkins's method
![image](https://user-images.githubusercontent.com/4150060/129654620-ebd90a6f-6090-4fe1-8536-f3213cbd8d76.png)

and headers of failure request.
![image](https://user-images.githubusercontent.com/4150060/129655039-32c21d70-a990-4936-b187-023d13729284.png)

**Reply by raspy8766** [reply]
> I took a failure log as similar as @rarkins's method
> ![image](https://user-images.githubusercontent.com/4150060/129654620-ebd90a6f-6090-4fe1-8536-f3213cbd8d76.png)
> 
> and headers of failure request.
> ![image](https://user-images.githubusercontent.com/4150060/129655039-32c21d70-a990-4936-b187-023d13729284.png)

Just a heads up @417-72KI you may want to invalidate and/or blackout your auth header there

**Comment by mark-raymond** [comment]
I am also hitting this problem (or at least the same symptoms), although I don't appear to have a CORS error in the network trace:

![image](https://user-images.githubusercontent.com/2951053/129722914-92d5272f-d735-4fb6-b359-6367b2eadeb8.png)

**Comment by hmps** [comment]
I had the same problem and was able to fix it by clearing all site data (Dev tools -> Application -> Clear site data) 🎉 

I hope that can be helpful to someone else!

**Reply by raspy8766** [reply]
Well, today it seems to be working...must be a browser caching issue like @hmps suggested
